pkg/chezmoi/archive.go

```diff
@@ -11,9 +11,11 @@ import (
 	"io"
 	"io/fs"
 	"path"
+	"path/filepath"
 	"strings"
 
 	"github.com/ulikunitz/xz"
+	"golang.org/x/exp/slices"
```

Comment:
This can be removed by using `map[string]struct{}` [as suggested](https://github.com/twpayne/chezmoi/pull/2699/files#r1072962956).